cli.ts

```diff
@@ -1835,14 +1835,18 @@ yargs(hideBin(process.argv)) // eslint-disable-line no-unused-expressions
     command: 'track-install',
     desc: false,
     handler: async (argv) => {
-      Avo.cliInstalled({
-        userId_: installIdOrUserId(),
-        cliInvokedByCi: invokedByCi(),
-      }).catch((error) => {
-        if (argv.verbose) {
-          console.error('Failed to track cli installed', error);
-        }
-      });
+      try {
+        Avo.cliInstalled({
```

Comment:
I was a bit surprised at first that this doesn't have an `await` but then I guess we're already `.catch`-ing that promise. I guess we're try-catching issues that happen before the called function manages to return the Promise?

Reply:
Probably yes, at least something is causing it to fail for people 🤔